docs/data/sql_functions_zh.yml

```
@@ -0,0 +1,820 @@
+    description: 以 64 个十六进制数字的字符串形式返回 string 的 SHA-256 哈希值；如果字符串为 NULL，则返回 NULL。
+  - sql: SHA384(string)
+    table: STRING.sha384()
+    description: 以 96 个十六进制数字的字符串形式返回 string 的 SHA-384 哈希值；如果字符串为 NULL，则返回 NULL。
+  - sql: SHA512(string)
+    table: STRING.sha512()
+    description: 以 128 个十六进制数字的字符串形式返回 string 的 SHA-512 哈希值；如果字符串为 NULL，则返回 NULL。
+  - sql: SHA2(string, hashLength)
+    table: STRING.sha2(INT)
+    description: |
+      使用 SHA-2 系列散列函数（SHA-224，SHA-256，SHA-384 或 SHA-512）返回散列。第一个参数字符串是要散列的字符串，第
+      二个参数 hashLength 是结果的位长（224，256，384 或 512）。如果 string 或 hashLength 为 NULL，则返回 NULL。
+
+auxilary:
+  - table:  callSql(STRING)
+    description: |
+      对 SQL 表达式的调用。给定的字符串在规划期间被解析并转换为 Table API 表达式。在运行时只计算翻译后的表达式。
+      目前，调用仅限于简单的标量表达式。不支持调用聚合函数或表值函数。子查询也是不允许的。
+      例如 `table.select(callSql("UPPER(myColumn)").substring(3))`
+  - table: ANY.as(NAME1, NAME2, ...)
+    description: 指定 ANY（字段）的名称。如果表达式扩展为多个字段，则可以指定其他名称。
+
+aggregate:
+  - sql: COUNT([ ALL ] expression | DISTINCT expression1 [, expression2]*)
+    description: 默认情况下或使用关键字 ALL，返回不为 NULL 的表达式的输入行数。使用 DISTINCT 则对所有值去重后计算。
+  - sql: COUNT(*) | COUNT(1)
+    table: FIELD.count
+    description: 返回输入行数。
+  - sql: AVG([ ALL | DISTINCT ] expression)
+    table: FIELD.avg
+    description: |
+      默认情况下或使用关键字 ALL，返回所有输入行中表达式的平均值（算术平均值）。使用 DISTINCT 则对所有值去重后计算。
+  - sql: SUM([ ALL | DISTINCT ] expression)
+    table: FIELD.sum
+    description: 默认情况下或使用关键字 ALL，返回所有输入行的表达式总和。使用 DISTINCT 则对所有值去重后计算。
+  - table: FIELD.sum0
+    description: 返回所有输入行的数字字段的总和。如果所有值都为 NULL，则返回 0。
+  - sql: MAX([ ALL | DISTINCT ] expression)
+    table: FIELD.max
+    description: 默认情况下或使用关键字 ALL，返回所有输入行中表达式的最大值。使用 DISTINCT 则对所有值去重后计算。
+  - sql: MIN([ ALL | DISTINCT ] expression )
+    table: FIELD.min
+    description: 默认情况下或使用关键字 ALL，返回所有输入行中表达式的最小值。使用 DISTINCT 则对所有值去重后计算。
+  - sql: STDDEV_POP([ ALL | DISTINCT ] expression)
+    table: FIELD.stddevPop
+    description: 默认情况下或使用关键字 ALL，返回所有输入行中表达式的总体标准偏差。使用 DISTINCT 则对所有值去重后计算。
+  - sql: STDDEV_SAMP([ ALL | DISTINCT ] expression)
+    table: FIELD.stddevSamp
+    description: 默认情况下或使用关键字 ALL，返回所有输入行中表达式的样本标准偏差。使用 DISTINCT 则对所有值去重后计算。
+  - sql: VAR_POP([ ALL | DISTINCT ] expression)
+    table: FIELD.varPop
+    description: |
+      默认情况下或使用关键字 ALL，返回所有输入行中表达式的总体方差（总体标准差的平方）。使用 DISTINCT 则对所有值去重后计算。
+  - sql: VAR_SAMP([ ALL | DISTINCT ] expression)
+    table: FIELD.varSamp
+    description: |
+      默认情况下或使用关键字 ALL，返回所有输入行中表达式的样本方差（样本标准差的平方）。使用 DISTINCT 则对所有值去重后计算。
+  - sql: COLLECT([ ALL | DISTINCT ] expression)
+    table: FIELD.collect
+    description: |
+      默认情况下或使用关键字 ALL，返回跨所有输入行的多组表达式。NULL 值将被忽略。使用 DISTINCT 则对所有值去重后计算。
+  - sql: VARIANCE([ ALL | DISTINCT ] expression)
+    description: VAR_SAMP() 的同义词。
+  - sql: RANK()
+    description: 返回值在一组值中的排名。结果是 1 加上分区顺序中当前行之前或等于当前行的行数。排名在序列中不一定连续。
+  - sql: DENSE_RANK()
+    description: |
+      返回值在一组值中的排名。结果是一加先前分配的等级值。与函数 rank 不同，dense_rank 不会在排名序列中产生间隙。
+  - sql: ROW_NUMER()
```

```suggestion
  - sql: ROW_NUMBER()
```

Same typo in the english part.